qiita_db/analysis.py

```diff
@@ -107,7 +108,8 @@ def get_by_status(cls, status):
         return {x[0] for x in conn_handler.execute_fetchall(sql, (status,))}
 
     @classmethod
-    def create(cls, owner, name, description, parent=None, from_default=False):
+    def create(cls, owner, name, description, portal, parent=None,
```

Comment:
I don't think you need to pass the portal. The portal should be automatically retrieved from the config file.